### PR TITLE
fixed external link redirect

### LINKED ISSUE #1210: External-Links - when accessed directly via. the URL do not redirect
there should be automatically redirect on this pages!

### PR DESCRIPTION
**Pull request by danrot**
__tasks:__

- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1210 
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php [hunk 1]
```diff
@@ -116,6 +116,14 @@ public function getRouteCollectionForRequest(Request $request)
                         )
                     );
                 } elseif (
+                    $content->getNodeType() === Structure::NODE_TYPE_EXTERNAL_LINK &&
+                    $content->getNodeState() === StructureInterface::STATE_PUBLISHED
+                ) {
+                    $collection->add(
+                        $content->getKey() . '_' . uniqid(),
+                        $this->getRedirectRoute($request, 'http://' . $content->getPropertyValue('external'))
```
Comment (dantleech) [inline src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php]: How do you know the property name is always going to be `external`? it depends on the form does it not?

Comment (dantleech) [inline src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php]: With the Document system you can just do `$document->getExternal()` or similar.

Comment (danrot) [inline src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php]: I guess the template which is used here is fixed, so this should work. Of course it's not the best solution to hardcode the value here, but that's something we can fix with your refactoring. Just wanted to add this feature without breaking any of the other stuff. Would like to change a lot more here...

Comment (wachterjohannes) [inline src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php]: i give @dantleech it is possible to "override" it! you should use `getPropertyByTag('sulu.rlp')->getValue()`

**Comment by wachterjohannes** [comment]
@danrot ACK i will wait for travis but then it could be merged

**Pull request merged**